cli/bpmetadata/cmd.go

```diff
@@ -203,3 +253,49 @@ func createContent(bpPath string, rootPath string, readmeContent []byte) Bluepri
 
 	return content
 }
+
+func writeMetadata(obj *BlueprintMetadata) error {
+	// marshal and write the file
+	yFile, err := yaml.Marshal(obj)
+	if err != nil {
+		return err
+	}
+
+	err = os.WriteFile(metadataFileName, yFile, 0644)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func unmarshalMetadata(f []byte) (*BlueprintMetadata, error) {
```

Comment:
We could include the file read in this fn too for parity with the writeMeta fn which marshals and writes.

Reply:
The file is already read and I wanted to uses that step for both checking status and reading together.

Reply:
I've added the file read logic within `unmarshalMetadata` now